Resources/Locale/en-US/emote/emote.ftl

```diff
@@ -0,0 +1 @@
+emote-blocked = You can't {$emote} right now!
```

Comment:
Localization for the message you get when you try to emote, but it's blocked.
Based on `You can't speak right now!" from Muted's implementation.